military-overlay/utils-source-and-test-data/schema-source-files/app6b/name_domains_values/Coded_Domain_LandEquipment.csv

```diff
@@ -0,0 +1,106 @@
+Name	Value
+GROUND TRACK EQUIPMENT	S-G-E-----
```

Comment:
This file doesn't look comma delimited (look like tabs) but apparently it still works?

Reply:
fixed with #367